crates/xilem_svg/src/common_attrs.rs

```diff
@@ -0,0 +1,165 @@
+// Copyright 2023 the Druid Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{any::Any, marker::PhantomData};
+
+use peniko::Brush;
+use xilem_core::{Id, MessageResult};
+
+use crate::{
+    context::{ChangeFlags, Cx},
+    view::{DomElement, View, ViewMarker},
+};
+
+pub struct Fill<T, V> {
+    child: V,
+    // This could reasonably be static Cow also, but keep things simple
```

Comment:
Not sure if this comment makes sense here (did it result from copy/paste from the `Class` view?) (Same for the `Stroke` below)